auth-web/src/views/auth/create-account/DuplicateAccountWarningView.vue

```diff
@@ -0,0 +1,127 @@
+              </div>
+            </div>
+            <div class="text-right">
+              <v-btn large color="primary" @click="navigateToRedirectUrl(org.id)" title="Go to Business Dashboard" data-test="goto-dashboard-button">Access Account</v-btn>
+            </div>
+          </v-card>
+
+        </v-col>
+    </v-row>
+      <v-row justify="center">
+      <v-col cols="12" sm="8" md="6"  class="text-center">
+        <v-btn large outlined color="primary" @click="createAccount()">
+            Create Another Account
+          </v-btn>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop, Vue } from 'vue-property-decorator'
+import { OrgWithAddress, Organization } from '@/models/Organization'
+import { Pages, SessionStorageKeys } from '@/util/constants'
+import { User, UserSettings } from '@/models/user'
+import { Address } from '@/models/address'
+import ConfigHelper from '@/util/config-helper'
+
+import { namespace } from 'vuex-class'
+
+const OrgModule = namespace('org')
+const UserModule = namespace('user')
+
+@Component({})
+export default class DuplicateAccountWarningView extends Vue {
+    @UserModule.Action('getUserSettings') private getUserSettings!: (keycloakGuid: string) => Promise<any>
+    // @UserModule.Action('getUserProfile') private getUserProfile!: (identifier: string) => Promise<User>
+    @UserModule.State('userProfile') private userProfile!: User
+    @OrgModule.Action('getOrgAdminContact') private getOrgAdminContact!: (orgId: number) => Promise<Address>
+    @OrgModule.State('currentOrganization') private currentOrganization!: Organization
+    @OrgModule.Action('addOrgSettings') private addOrgSettings!: (currentOrganization: Organization) => Promise<UserSettings>
+    private orgsOfUser: OrgWithAddress[] = []
+    private isLoading: boolean = false
+    @Prop({ default: '' }) redirectToUrl !: string
+
+    private async mounted () {
+      try {
+        this.isLoading = true
+        await this.addOrgSettings(this.currentOrganization)
+        const userSettings: UserSettings[] = await this.getUserSettings(this.userProfile?.keycloakGuid)
+        for (let i = 0; i < userSettings.length && userSettings.length > 0; i++) {
+          const orgId = parseInt(userSettings[i].id)
+          const orgAdminContact = await this.getOrgAdminContact(orgId)
+          const orgOfUser: OrgWithAddress = {
+            id: orgId,
+            name: userSettings[i].label,
+            addressLine: `${orgAdminContact.street} ${orgAdminContact.city} ${orgAdminContact.region} ${orgAdminContact.postalCode} ${orgAdminContact.country}`
+          }
+          this.orgsOfUser.push(orgOfUser)
+        }
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.log(`Error while loading duplicate accounts ${err}`)
+      } finally {
+        this.isLoading = false
+      }
+    }
+
+    private navigateToRedirectUrl (accountId: number): void {
```

Comment:
account id not used

Reply:
@saravanpa-aot  we are syncing org and org settings, is this fine ?